Automattic/sensei
# Add course list icon

Implements https://github.com/Automattic/sensei/issues/5588

### Changes proposed in this Pull Request

* Added an icon for course list block

### Testing instructions

- Add a Course List block on GB editor
- While doing that, make sure the icon in the inserter is the one from the attached screenshot
- The same icon should be shown in the block and the tree

### Screenshot / Video

![image](https://user-images.githubusercontent.com/6820724/188779715-064ba3cc-4c52-4fff-a18e-72ab9afc1d90.png)
